src/assets/sass/protocol/components/_navigation.scss

```
@@ -50,6 +50,34 @@
     }
 }
 
+/* -------------------------------------------------------------------------- */
+// Sticky navigation styles
+
+@supports (position: sticky) {
+    html.mzp-has-sticky-navigation {
+        .mzp-c-navigation.mzp-is-sticky {
+            @include transition(transform 300ms ease-in-out);
+            left: 0;
+            position: sticky;
+            top: 0;
+            z-index: 1000;
+
+            &.mzp-is-scrolling {
+                box-shadow: 0 0 5px 1px rgba(9, 32, 77, .12);
```

We have tokens for box-shadows, though they're bigger than this and spread in all directions so I don't think a token can work here. But maybe we should at least match the color and layering of other shadows.

```suggestion
                // Shadow colors are equivalent to $color-ink-90, $color-blue-90, $color-ink-90
                box-shadow: 0 0 6px 1px rgba(29, 17, 51, .04), 0 0 8px 2px rgba(9, 32, 77, .12), 0 0 5px -3px rgba(29, 17, 51, .12);
```